crossfit/op/tokenize.py

```diff
@@ -61,6 +62,8 @@ def tokenize_strings(self, sentences, max_length=None):
             self.padding_side = tokenizer.padding_side
             self.pad_token_id = tokenizer.pad_token_id
 
+            if isinstance(sentences, pd.Series):
+                sentences = cudf.from_pandas(sentences)
             if isinstance(sentences, cudf.Series):
                 sentences = sentences.to_arrow().to_pylist()
```

Comment:
How about we do something like below ? 

```suggestion
            if isinstance(sentences, cudf.Series):
                sentences = sentences.to_arrow().to_pylist()
            elif isinstance(sentences, pd.Series):
                sentences = sentences.to_list()
```